darts/timeseries.py

```diff
@@ -2438,94 +2439,97 @@ def with_columns_renamed(
+        """Simple wrapper around :func:`np.ndarray.min()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = self._xa.values.min(axis=2, keepdims=True)
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def max(
-        self, axis=None, skipna=True, level=None, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.max()` for deterministic series."""
-        return self.pd_dataframe(copy=False).max(
-            axis, skipna, level, numeric_only, **kwargs
-        )
+    def max(self) -> "TimeSeries":
+        """Simple wrapper around :func:`np.ndarray.max()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = self._xa.values.max(axis=2, keepdims=True)
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def sum(
-        self,
-        axis=None,
-        skipna=None,
-        level=None,
-        numeric_only=None,
-        min_count=0,
-        **kwargs,
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.sum()` for deterministic series."""
-        return self.pd_dataframe(copy=False).sum(
-            axis, skipna, level, numeric_only, min_count, **kwargs
-        )
+    def sum(self) -> "TimeSeries":
+        """Simple wrapper around :func:`np.ndarray.sum()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = self._xa.values.sum(axis=2, keepdims=True)
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def median(
-        self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.median()` for deterministic series."""
-        return self.pd_dataframe(copy=False).median(
-            axis, skipna, level, numeric_only, **kwargs
+    def median(self) -> "TimeSeries":
+        """Simple wrapper around :func:`np.median()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = np.median(
+            self._xa.values, axis=2, out=None, overwrite_input=False, keepdims=True
         )
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def autocorr(self, lag=1) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.autocorr()` for deterministic series."""
-        return self.pd_dataframe(copy=False).autocorr(lag)
-
-    def describe(self, percentiles=None, include=None, exclude=None) -> pd.DataFrame:
-        """Simple wrapper around :func:`pd.DataFrame.describe()` for deterministic series."""
-        return self.pd_dataframe(copy=False).describe(percentiles, include, exclude)
+    def quantile(self, q: float, method: str = "linear") -> "TimeSeries":
```

Comment:
We already have a function `quantile_timeseries` which does the same thing :)
However uses a dataframe which is not great (I prefer using Numpy), and so maybe we could leverage your implementation there? And also keep the `quantile()` method too, as:
```
def quantile(...):
    return self.quantile_timeseries(...)
```